Jenkinsfile

```diff
@@ -133,6 +133,8 @@ if (env.CHANGE_TITLE && env.CHANGE_TITLE.contains('[ci-skip]')) {
                                             sh "npm run vendor-install"
                                             sh "unzip ${env.KEYBASE_JS_VENDOR_DIR}/flow/flow-linux64*.zip -d ${env.BASEDIR}"
                                             sh "${env.BASEDIR}/flow/flow status shared"
+                                            sh "npm run shrinkwrap-audit"
```

Comment:
Obviously if this is the case, `shrinkwrap-audit.js` will have to be in the root directory instead of desktop.